examples/medplum-provider/src/components/DoseSpotIcon.tsx

```diff
@@ -91,7 +20,7 @@ export function DoseSpotIcon(): JSX.Element {
     </Tooltip>
   );
 
-  return unreadCount > 0 ? (
+  return unreadCount !== undefined && unreadCount > 0 ? (
```

Comment:
```suggestion
  return unreadCount ? (
```
Think we can simplify this safely, right?

Reply:
✅